app/src/main/java/com/jalasoft/routesapp/data/model/local/RouteDetail.kt

```diff
@@ -19,14 +19,15 @@ data class RouteDetail(
         fun getRouteDetailFromLocationList(
             routeName: String,
             list: List<Location>,
-            icon: LineCategoryIcons = LineCategoryIcons(),
+            whiteIcon: String,
+            blackIcon: String,
             category: String = "",
             averageVelocity: Double = Constants.AVG_WALKING_PACE,
             walkDirection: WalkDirection = WalkDirection.IS_NOT_WALKING
         ): RouteDetail {
             val distance = GoogleMapsHelper.getLocationListDistance(list)
-            val estimatedTime = distance / averageVelocity
-            return RouteDetail(routeName, category, icon, estimatedTime.roundToInt(), distance.roundToInt(), walkDirection)
+            val estimatedTime = distance / averageVelocity / 60
```

Comment:
add parentheses to add clarity to this formula, also comment next to the number the meaning of that 60

Reply:
Added comments and parentheses to the formula to get the estimated time.